deploy/PRODUCTION.md

```
@@ -29,20 +29,111 @@ On the production server:
+sudo ufw allow 80/tcp   # HTTP
+sudo ufw allow 443/tcp  # HTTPS
+
+# Enable firewall
+sudo ufw --force enable
+
+# Verify firewall status
+sudo ufw status
+```
+
+Expected output should show:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)